src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -133,6 +133,14 @@ public void addTask(Task t) {
         tasks.add(t);
     }
 
+    /**
+     * Removes {@code key} from this {@code AddressBook}.
+     * {@code key} must exist in the address book.
+     */
+    public void removeTask(Task key) {
```

Comment:
i think @freshcabbage123 mentioned standardize to delete
```suggestion
    public void deleteTask(Task key) {
```